packages/portal/src/pages/media/index.vue

```diff
@@ -49,8 +49,10 @@
 
     computed: {
       manifest() {
-        return `${this.$config.europeana.apis.iiifPresentation.url}${this.id}/manifest?format=3`;
+        const recordEndpointParam = this.$config.europeana.apis.record.url ? '&recordApi=' + this.$config.europeana.apis.record.url.replace('/api/v2', '') : '';
```

Comment:
This will not always result in the correct recordUrl parameter.

Our IIIF Presentation API expects recordUrl to be just the base URL with no path, i.e. the origin. If portal.js is configured with a Record API URL of https://api.acceptance.eanadev.org/record, then this replace will not remove the path, resulting in a failing IIIF manifest URL of e.g. https://iiif.acceptance.eanadev.org/presentation/170/data_euscreenXL_EUS_67F3ED0D435191E8A739326CD5F5F9F8/manifest?format=3&recordApi=https://api.acceptance.eanadev.org/record

Instead, use `new URL(this.$config.europeana.apis.record.url).origin`